### PR TITLE
adding ICSP config utilities

### PR DESCRIPTION
**Pull request by omrirh**
Adding support for ImageContentSourcePolicy creation

### CODE REVIEW

File: ocp_utilities/infra.py [hunk 1]
```diff
@@ -288,3 +292,58 @@ def cluster_resource(base_class):
...
+    folder_name, image, source_url, pull_secret=None, filter_options=""
+):
+    base_command = create_icsp_command(
+        image=image,
+        source_url=source_url,
+        folder_name=folder_name,
+        pull_secret=pull_secret,
+        filter_options=filter_options,
+    )
+    rc, _, _ = run_command(
```
Comment (myakove) [inline ocp_utilities/infra.py:332]:
```
assert run_command(
        command=shlex.split(base_command),
        verify_stderr=False,
    )[0]
```

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_utilities/infra.py (hunk 1 above)
Comment (omrirh) [reply ocp_utilities/infra.py]: done

**Review by myakove — DISMISSED** [review]

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Comment by myakove** [comment]
depends on https://github.com/RedHatQE/openshift-python-wrapper/pull/1211

**Comment by rnetser** [comment]
@omrirh this PR requires https://github.com/RedHatQE/openshift-python-wrapper/pull/1211 to be merged and cherrypicked; marking this PR as `verified` (and thus leading to its merge) will lead to broken code. Please remove.

**Comment by omrirh** [comment]
/~verified

**Review by rnetser — APPROVED** [review]

**Pull request merged**